tr/cs-221-reflex-models.md

```diff
@@ -0,0 +1,541 @@
+**Reflex-based models translation** [[webpage]](https://stanford.edu/~shervine/teaching/cs-221/cheatsheet-reflex-models)
+
+<br>
+
+**1. Reflex-based models with Machine Learning**
+
+&#10230; Makine Öğrenmesi ile Refleks tabanlı modeller
+
+<br>
+
+
+**2. Linear predictors**
+
+&#10230; Doğrusal öngörücüler
+
+<br>
+
+
+**3. In this section, we will go through reflex-based models that can improve with experience, by going through samples that have input-output pairs.**
+
+&#10230; Bu bölümde, girdi-çıktı çiftleri olan örneklerden geçerek, deneyim ile gelişebilecek refleks bazlı modelleri göreceğiz.
+
+<br>
+
+
+**4. Feature vector ― The feature vector of an input x is noted ϕ(x) and is such that:**
+
+&#10230; Öznitelik vektörü - x girişinin öznitelik vektörü ϕ (x) olarak not edilir ve şöyledir:
+
+<br>
+
+
+**5. Score ― The score s(x,w) of an example (ϕ(x),y)∈Rd×R associated to a linear model of weights w∈Rd is given by the inner product:**
+
+&#10230; Puan - Bir örneğin s(x, w)si ni (ϕ(x),y))∈Rd×R, w∈Rd doğrusal ağırlık modeline bağlı olarak:
+
+<br>
+
+
+**6. Classification**
+
+&#10230; Sınıflandırma
+
+<br>
+
+
+**7. Linear classifier ― Given a weight vector w∈Rd and a feature vector ϕ(x)∈Rd, the binary linear classifier fw is given by:**
+
+&#10230; Doğrusal sınıflandırıcı - Bir ağırlık vektörü w∈Rd ve bir öznitelik vektörü ϕ(x)∈Rd verildiğinde, ikili doğrusal sınıflandırıcı fw şöyle verilir:
+
+<br>
+
+
+**8. if**
+
+&#10230;
+
+<br> Eğer
+
+
+**9. Margin ― The margin m(x,y,w)∈R of an example (ϕ(x),y)∈Rd×{−1,+1} associated to a linear model of weights w∈Rd quantifies the confidence of the prediction: larger values are better. It is given by:**
```

Comment:
Marj ― (ϕ (x), y) ∈Rd×{−1,+1} örneğinin m(x,y,w)∈R marjları w∈Rd doğrusal ağırlık modeliyle ilişkili olarak, tahminin güvenirliği ölçülür: daha büyük değerler daha iyidir. Tarafından verilir:

Reply:
Bunu tamamen atlamışım. "It is given by:"ı diğer maddelerdeki gibi "şöyle ifade edilir", "şöyle verilir" şeklinde çevirdim.